diff --git a/contracts/rewards/RewardsDistributor.sol b/contracts/rewards/RewardsDistributor.sol
index 04d057a..f3df1ff 100644
--- a/contracts/rewards/RewardsDistributor.sol
+++ b/contracts/rewards/RewardsDistributor.sol
@@ -423,7 +423,8 @@ abstract contract RewardsDistributor is IRewardsDistributor {
     RewardsDataTypes.UserAssetBalance[] memory userAssetBalances
   ) internal view returns (uint256 unclaimedRewards) {
     // Add unrealized rewards
-    for (uint256 i = 0; i < userAssetBalances.length; i++) {
+    /// SwapArgumentsOperatorMutation(`i < userAssetBalances.length` |==> `userAssetBalances.length < i`) of: `for (uint256 i = 0; i < userAssetBalances.length; i++) {`
+    for (uint256 i = 0; userAssetBalances.length < i; i++) {
       if (userAssetBalances[i].userBalance == 0) {
         unclaimedRewards += _assets[userAssetBalances[i].asset]
           .rewards[reward]
