diff --git a/contracts/rewards/RewardsDistributor.sol b/contracts/rewards/RewardsDistributor.sol
index 04d057a..b5dad1e 100644
--- a/contracts/rewards/RewardsDistributor.sol
+++ b/contracts/rewards/RewardsDistributor.sol
@@ -194,7 +194,8 @@ abstract contract RewardsDistributor is IRewardsDistributor {
     address[] calldata rewards,
     uint88[] calldata newEmissionsPerSecond
   ) external override onlyEmissionManager {
-    require(rewards.length == newEmissionsPerSecond.length, 'INVALID_INPUT');
+    /// RequireMutation(`rewards.length == newEmissionsPerSecond.length` |==> `true`) of: `require(rewards.length == newEmissionsPerSecond.length, 'INVALID_INPUT');`
+    require(true, 'INVALID_INPUT');
     for (uint256 i = 0; i < rewards.length; i++) {
       RewardsDataTypes.AssetData storage assetConfig = _assets[asset];
       RewardsDataTypes.RewardData storage rewardConfig = _assets[asset].rewards[rewards[i]];
