diff --git a/contracts/rewards/RewardsDistributor.sol b/contracts/rewards/RewardsDistributor.sol
index 04d057a..ae5d737 100644
--- a/contracts/rewards/RewardsDistributor.sol
+++ b/contracts/rewards/RewardsDistributor.sol
@@ -231,7 +231,8 @@ abstract contract RewardsDistributor is IRewardsDistributor {
    **/
   function _configureAssets(RewardsDataTypes.RewardsConfigInput[] memory rewardsInput) internal {
     for (uint256 i = 0; i < rewardsInput.length; i++) {
-      if (_assets[rewardsInput[i].asset].decimals == 0) {
+      /// IfStatementMutation(`_assets[rewardsInput[i].asset].decimals == 0` |==> `false`) of: `if (_assets[rewardsInput[i].asset].decimals == 0) {`
+      if (false) {
         //never initialized before, adding to the list of assets
         _assetsList.push(rewardsInput[i].asset);
       }
