diff --git a/contracts/rewards/RewardsDistributor.sol b/contracts/rewards/RewardsDistributor.sol
index 04d057a..1581740 100644
--- a/contracts/rewards/RewardsDistributor.sol
+++ b/contracts/rewards/RewardsDistributor.sol
@@ -249,7 +249,8 @@ abstract contract RewardsDistributor is IRewardsDistributor {
         _assets[rewardsInput[i].asset].availableRewards[
           _assets[rewardsInput[i].asset].availableRewardsCount
         ] = rewardsInput[i].reward;
-        _assets[rewardsInput[i].asset].availableRewardsCount++;
+        /// DeleteExpressionMutation(`_assets[rewardsInput[i].asset].availableRewardsCount++` |==> `assert(true)`) of: `_assets[rewardsInput[i].asset].availableRewardsCount++;`
+        assert(true);
       }
 
       // Add reward address to global rewards list if still not enabled
