diff --git a/contracts/rewards/RewardsDistributor.sol b/contracts/rewards/RewardsDistributor.sol
index 04d057a..97a3150 100644
--- a/contracts/rewards/RewardsDistributor.sol
+++ b/contracts/rewards/RewardsDistributor.sol
@@ -262,7 +262,8 @@ abstract contract RewardsDistributor is IRewardsDistributor {
       (uint256 newIndex, ) = _updateRewardData(
         rewardConfig,
         rewardsInput[i].totalSupply,
-        10**decimals
+        /// BinaryOpMutation(`**` |==> `*`) of: `10**decimals`
+        10*decimals
       );
 
       // Configure emission and distribution end of the reward per asset
