diff --git a/contracts/rewards/RewardsDistributor.sol b/contracts/rewards/RewardsDistributor.sol
index 04d057a..4679332 100644
--- a/contracts/rewards/RewardsDistributor.sol
+++ b/contracts/rewards/RewardsDistributor.sol
@@ -262,7 +262,8 @@ abstract contract RewardsDistributor is IRewardsDistributor {
       (uint256 newIndex, ) = _updateRewardData(
         rewardConfig,
         rewardsInput[i].totalSupply,
-        10**decimals
+        /// SwapArgumentsOperatorMutation(`10**decimals` |==> `decimals ** 10`) of: `10**decimals`
+        decimals ** 10
       );
 
       // Configure emission and distribution end of the reward per asset
