diff --git a/contracts/rewards/RewardsDistributor.sol b/contracts/rewards/RewardsDistributor.sol
index 04d057a..6b7d147 100644
--- a/contracts/rewards/RewardsDistributor.sol
+++ b/contracts/rewards/RewardsDistributor.sol
@@ -298,7 +298,8 @@ abstract contract RewardsDistributor is IRewardsDistributor {
   ) internal returns (uint256, bool) {
     (uint256 oldIndex, uint256 newIndex) = _getAssetIndex(rewardData, totalSupply, assetUnit);
     bool indexUpdated;
-    if (newIndex != oldIndex) {
+    /// IfStatementMutation(`newIndex != oldIndex` |==> `true`) of: `if (newIndex != oldIndex) {`
+    if (true) {
       require(newIndex <= type(uint104).max, 'INDEX_OVERFLOW');
       indexUpdated = true;
 
