diff --git a/contracts/rewards/RewardsDistributor.sol b/contracts/rewards/RewardsDistributor.sol
index 04d057a..4e15482 100644
--- a/contracts/rewards/RewardsDistributor.sol
+++ b/contracts/rewards/RewardsDistributor.sol
@@ -306,7 +306,8 @@ abstract contract RewardsDistributor is IRewardsDistributor {
       rewardData.index = uint104(newIndex);
       rewardData.lastUpdateTimestamp = block.timestamp.toUint32();
     } else {
-      rewardData.lastUpdateTimestamp = block.timestamp.toUint32();
+      /// DeleteExpressionMutation(`rewardData.lastUpdateTimestamp = block.timestamp.toUint32()` |==> `assert(true)`) of: `rewardData.lastUpdateTimestamp = block.timestamp.toUint32();`
+      assert(true);
     }
 
     return (newIndex, indexUpdated);
