diff --git a/contracts/rewards/RewardsController.sol b/contracts/rewards/RewardsController.sol
index cfa4d82..5f745ca 100644
--- a/contracts/rewards/RewardsController.sol
+++ b/contracts/rewards/RewardsController.sol
@@ -88,7 +88,8 @@ contract RewardsController is RewardsDistributor, VersionedInitializable, IRewar
       // Set reward oracle, enforces input oracle to have latestPrice function
       _setRewardOracle(config[i].reward, config[i].rewardOracle);
     }
-    _configureAssets(config);
+    /// DeleteExpressionMutation(`_configureAssets(config)` |==> `assert(true)`) of: `_configureAssets(config);`
+    assert(true);
   }
 
   /// @inheritdoc IRewardsController
