diff --git a/contracts/rewards/RewardsController.sol b/contracts/rewards/RewardsController.sol
index cfa4d82..9bd6950 100644
--- a/contracts/rewards/RewardsController.sol
+++ b/contracts/rewards/RewardsController.sol
@@ -123,7 +123,8 @@ contract RewardsController is RewardsDistributor, VersionedInitializable, IRewar
     address to,
     address reward
   ) external override returns (uint256) {
-    require(to != address(0), 'INVALID_TO_ADDRESS');
+    /// RequireMutation(`to != address(0)` |==> `true`) of: `require(to != address(0), 'INVALID_TO_ADDRESS');`
+    require(true, 'INVALID_TO_ADDRESS');
     return _claimRewards(assets, amount, msg.sender, msg.sender, to, reward);
   }
 
