diff --git a/contracts/rewards/RewardsController.sol b/contracts/rewards/RewardsController.sol
index cfa4d82..db442a0 100644
--- a/contracts/rewards/RewardsController.sol
+++ b/contracts/rewards/RewardsController.sol
@@ -155,7 +155,8 @@ contract RewardsController is RewardsDistributor, VersionedInitializable, IRewar
     override
     returns (address[] memory rewardsList, uint256[] memory claimedAmounts)
   {
-    require(to != address(0), 'INVALID_TO_ADDRESS');
+    /// RequireMutation(`to != address(0)` |==> `false`) of: `require(to != address(0), 'INVALID_TO_ADDRESS');`
+    require(false, 'INVALID_TO_ADDRESS');
     return _claimAllRewards(assets, msg.sender, msg.sender, to);
   }
 
