diff --git a/contracts/rewards/RewardsController.sol b/contracts/rewards/RewardsController.sol
index cfa4d82..2c9f88a 100644
--- a/contracts/rewards/RewardsController.sol
+++ b/contracts/rewards/RewardsController.sol
@@ -240,7 +240,8 @@ contract RewardsController is RewardsDistributor, VersionedInitializable, IRewar
       address asset = assets[i];
       totalRewards += _assets[asset].rewards[reward].usersData[user].accrued;
 
-      if (totalRewards <= amount) {
+      /// SwapArgumentsOperatorMutation(`totalRewards <= amount` |==> `amount <= totalRewards`) of: `if (totalRewards <= amount) {`
+      if (amount <= totalRewards) {
         _assets[asset].rewards[reward].usersData[user].accrued = 0;
       } else {
         uint256 difference = totalRewards - amount;
