diff --git a/contracts/rewards/RewardsController.sol b/contracts/rewards/RewardsController.sol
index cfa4d82..0742159 100644
--- a/contracts/rewards/RewardsController.sol
+++ b/contracts/rewards/RewardsController.sol
@@ -238,7 +238,8 @@ contract RewardsController is RewardsDistributor, VersionedInitializable, IRewar
     _updateDataMultiple(user, _getUserAssetBalances(assets, user));
     for (uint256 i = 0; i < assets.length; i++) {
       address asset = assets[i];
-      totalRewards += _assets[asset].rewards[reward].usersData[user].accrued;
+      /// AssignmentMutation(`_assets[asset].rewards[reward].usersData[user].accrued` |==> `1`) of: `totalRewards += _assets[asset].rewards[reward].usersData[user].accrued;`
+      totalRewards += 1;
 
       if (totalRewards <= amount) {
         _assets[asset].rewards[reward].usersData[user].accrued = 0;
