diff --git a/contracts/rewards/RewardsController.sol b/contracts/rewards/RewardsController.sol
index cfa4d82..1943b66 100644
--- a/contracts/rewards/RewardsController.sol
+++ b/contracts/rewards/RewardsController.sol
@@ -245,7 +245,8 @@ contract RewardsController is RewardsDistributor, VersionedInitializable, IRewar
       } else {
         uint256 difference = totalRewards - amount;
         totalRewards -= difference;
-        _assets[asset].rewards[reward].usersData[user].accrued = difference.toUint128();
+        /// DeleteExpressionMutation(`_assets[asset].rewards[reward].usersData[user].accrued = difference.toUint128()` |==> `assert(true)`) of: `_assets[asset].rewards[reward].usersData[user].accrued = difference.toUint128();`
+        assert(true);
         break;
       }
     }
