diff --git a/contracts/rewards/RewardsController.sol b/contracts/rewards/RewardsController.sol
index cfa4d82..04205eb 100644
--- a/contracts/rewards/RewardsController.sol
+++ b/contracts/rewards/RewardsController.sol
@@ -282,7 +282,8 @@ contract RewardsController is RewardsDistributor, VersionedInitializable, IRewar
 
     _updateDataMultiple(user, _getUserAssetBalances(assets, user));
 
-    for (uint256 i = 0; i < assets.length; i++) {
+    /// SwapArgumentsOperatorMutation(`i < assets.length` |==> `assets.length < i`) of: `for (uint256 i = 0; i < assets.length; i++) {`
+    for (uint256 i = 0; assets.length < i; i++) {
       address asset = assets[i];
       for (uint256 j = 0; j < rewardsListLength; j++) {
         if (rewardsList[j] == address(0)) {
