diff --git a/contracts/rewards/RewardsController.sol b/contracts/rewards/RewardsController.sol
index cfa4d82..bedf0b8 100644
--- a/contracts/rewards/RewardsController.sol
+++ b/contracts/rewards/RewardsController.sol
@@ -317,7 +317,8 @@ contract RewardsController is RewardsDistributor, VersionedInitializable, IRewar
 
     bool success = transferStrategy.performTransfer(to, reward, amount);
 
-    require(success == true, 'TRANSFER_ERROR');
+    /// RequireMutation(`success == true` |==> `false`) of: `require(success == true, 'TRANSFER_ERROR');`
+    require(false, 'TRANSFER_ERROR');
   }
 
   /**
