diff --git a/contracts/rewards/RewardsController.sol b/contracts/rewards/RewardsController.sol
index cfa4d82..b8e6400 100644
--- a/contracts/rewards/RewardsController.sol
+++ b/contracts/rewards/RewardsController.sol
@@ -295,7 +295,8 @@ contract RewardsController is RewardsDistributor, VersionedInitializable, IRewar
         }
       }
     }
-    for (uint256 i = 0; i < rewardsListLength; i++) {
+    /// UnaryOperatorMutation(`++` |==> `--`) of: `for (uint256 i = 0; i < rewardsListLength; i++) {`
+    for (uint256 i = 0; i < rewardsListLength; i--) {
       _transferRewards(to, rewardsList[i], claimedAmounts[i]);
       emit RewardsClaimed(user, rewardsList[i], to, claimer, claimedAmounts[i]);
     }
