diff --git a/contracts/rewards/RewardsDistributor.sol b/contracts/rewards/RewardsDistributor.sol
index 04d057a..5682d6d 100644
--- a/contracts/rewards/RewardsDistributor.sol
+++ b/contracts/rewards/RewardsDistributor.sol
@@ -499,7 +499,7 @@ abstract contract RewardsDistributor is IRewardsDistributor {
     uint256 totalSupply,
     uint256 assetUnit
   ) internal view returns (uint256, uint256) {
-    uint256 oldIndex = rewardData.index;
+    uint256 oldIndex = rewardData.distributionEnd;
     uint256 distributionEnd = rewardData.distributionEnd;
     uint256 emissionPerSecond = rewardData.emissionPerSecond;
     uint256 lastUpdateTimestamp = rewardData.lastUpdateTimestamp;
