diff --git a/contracts/rewards/RewardsDistributor.sol b/contracts/rewards/RewardsDistributor.sol
index 04d057a..dcd3a86 100644
--- a/contracts/rewards/RewardsDistributor.sol
+++ b/contracts/rewards/RewardsDistributor.sol
@@ -335,7 +335,7 @@ abstract contract RewardsDistributor is IRewardsDistributor {
       // already checked for overflow in _updateRewardData
       rewardData.usersData[user].index = uint104(newAssetIndex);
       if (userBalance != 0) {
-        rewardsAccrued = _getRewards(userBalance, newAssetIndex, userIndex, assetUnit);
+        rewardsAccrued = _getRewards(userBalance, newAssetIndex, assetUnit, userIndex);
 
         rewardData.usersData[user].accrued += rewardsAccrued.toUint128();
       }
