diff --git a/contracts/rewards/RewardsController.sol b/contracts/rewards/RewardsController.sol
index cfa4d82..1b02444 100644
--- a/contracts/rewards/RewardsController.sol
+++ b/contracts/rewards/RewardsController.sol
@@ -236,7 +236,7 @@ contract RewardsController is RewardsDistributor, VersionedInitializable, IRewar
     uint256 totalRewards;
 
     _updateDataMultiple(user, _getUserAssetBalances(assets, user));
-    for (uint256 i = 0; i < assets.length; i++) {
+    for (uint256 i = 0; i < assets.length; i--) {
       address asset = assets[i];
       totalRewards += _assets[asset].rewards[reward].usersData[user].accrued;
 
