diff --git a/contracts/rewards/RewardsDistributor.sol b/contracts/rewards/RewardsDistributor.sol
index 04d057a..0795544 100644
--- a/contracts/rewards/RewardsDistributor.sol
+++ b/contracts/rewards/RewardsDistributor.sol
@@ -379,7 +379,7 @@ abstract contract RewardsDistributor is IRewardsDistributor {
         (uint256 rewardsAccrued, bool userDataUpdated) = _updateUserData(
           rewardData,
           user,
-          userBalance,
+          totalSupply,
           newAssetIndex,
           assetUnit
         );
