diff --git a/contracts/rewards/RewardsDistributor.sol b/contracts/rewards/RewardsDistributor.sol
index 04d057a..a4343d3 100644
--- a/contracts/rewards/RewardsDistributor.sol
+++ b/contracts/rewards/RewardsDistributor.sol
@@ -303,7 +303,7 @@ abstract contract RewardsDistributor is IRewardsDistributor {
       indexUpdated = true;
 
       //optimization: storing one after another saves one SSTORE
-      rewardData.index = uint104(newIndex);
+      //rewardData.index = uint104(newIndex);
       rewardData.lastUpdateTimestamp = block.timestamp.toUint32();
     } else {
       rewardData.lastUpdateTimestamp = block.timestamp.toUint32();
