diff --git a/contracts/rewards/RewardsController.sol b/contracts/rewards/RewardsController.sol
index cfa4d82..c9961ba 100644
--- a/contracts/rewards/RewardsController.sol
+++ b/contracts/rewards/RewardsController.sol
@@ -47,7 +47,7 @@ contract RewardsController is RewardsDistributor, VersionedInitializable, IRewar
    * @dev Initialize for RewardsController
    * @dev It expects an address as argument since its initialized via PoolAddressesProvider._updateImpl()
    **/
-  function initialize(address) external initializer {}
+  function initialize(address) external {}
 
   /// @inheritdoc IRewardsController
   function getClaimer(address user) external view override returns (address) {
