diff --git a/contracts/rewards/RewardsDistributor.sol b/contracts/rewards/RewardsDistributor.sol
index 04d057a..98da3c8 100644
--- a/contracts/rewards/RewardsDistributor.sol
+++ b/contracts/rewards/RewardsDistributor.sol
@@ -431,7 +431,7 @@ abstract contract RewardsDistributor is IRewardsDistributor {
           .accrued;
       } else {
         unclaimedRewards +=
-          _getPendingRewards(user, reward, userAssetBalances[i]) +
+          _getPendingRewards(user, reward, userAssetBalances[i]) -
           _assets[userAssetBalances[i].asset].rewards[reward].usersData[user].accrued;
       }
     }
