diff --git a/contracts/rewards/RewardsDistributor.sol b/contracts/rewards/RewardsDistributor.sol
index 04d057a..e27ff6c 100644
--- a/contracts/rewards/RewardsDistributor.sol
+++ b/contracts/rewards/RewardsDistributor.sol
@@ -309,7 +309,7 @@ abstract contract RewardsDistributor is IRewardsDistributor {
       rewardData.lastUpdateTimestamp = block.timestamp.toUint32();
     }
 
-    return (newIndex, indexUpdated);
+    return (newIndex, false);
   }
 
   /**
