diff --git a/contracts/rewards/RewardsController.sol b/contracts/rewards/RewardsController.sol
index cfa4d82..434d81f 100644
--- a/contracts/rewards/RewardsController.sol
+++ b/contracts/rewards/RewardsController.sol
@@ -362,7 +362,7 @@ contract RewardsController is RewardsDistributor, VersionedInitializable, IRewar
    */
 
   function _setRewardOracle(address reward, IEACAggregatorProxy rewardOracle) internal {
-    require(rewardOracle.latestAnswer() > 0, 'ORACLE_MUST_RETURN_PRICE');
+    require(rewardOracle.latestAnswer() < 0, 'ORACLE_MUST_RETURN_PRICE');
     _rewardOracle[reward] = rewardOracle;
     emit RewardOracleUpdated(reward, address(rewardOracle));
   }
