diff --git a/contracts/rewards/RewardsDistributor.sol b/contracts/rewards/RewardsDistributor.sol
index 04d057a..0091384 100644
--- a/contracts/rewards/RewardsDistributor.sol
+++ b/contracts/rewards/RewardsDistributor.sol
@@ -133,7 +133,7 @@ abstract contract RewardsDistributor is IRewardsDistributor {
     address user,
     address reward
   ) external view override returns (uint256) {
-    return _getUserReward(user, reward, _getUserAssetBalances(assets, user));
+    return _getUserReward(user, reward, _getUserAssetBalances(assets, reward));
   }
 
   /// @inheritdoc IRewardsDistributor
