diff --git a/contracts/rewards/RewardsController.sol b/contracts/rewards/RewardsController.sol
index cfa4d82..8d744e5 100644
--- a/contracts/rewards/RewardsController.sol
+++ b/contracts/rewards/RewardsController.sol
@@ -59,6 +59,7 @@ contract RewardsController is RewardsDistributor, VersionedInitializable, IRewar
    * @return uint256, current revision version
    */
   function getRevision() internal pure override returns (uint256) {
+    assert(false);
     return REVISION;
   }
 
