diff --git a/contracts/rewards/RewardsDistributor.sol b/contracts/rewards/RewardsDistributor.sol
index 04d057a..8f9b0f0 100644
--- a/contracts/rewards/RewardsDistributor.sol
+++ b/contracts/rewards/RewardsDistributor.sol
@@ -245,7 +245,7 @@ abstract contract RewardsDistributor is IRewardsDistributor {
       ];
 
       // Add reward address to asset available rewards if latestUpdateTimestamp is zero
-      if (rewardConfig.lastUpdateTimestamp == 0) {
+      if (rewardConfig.lastUpdateTimestamp != 0) {
         _assets[rewardsInput[i].asset].availableRewards[
           _assets[rewardsInput[i].asset].availableRewardsCount
         ] = rewardsInput[i].reward;
