diff --git a/contracts/rewards/RewardsController.sol b/contracts/rewards/RewardsController.sol
index cfa4d82..4c2092e 100644
--- a/contracts/rewards/RewardsController.sol
+++ b/contracts/rewards/RewardsController.sol
@@ -113,7 +113,7 @@ contract RewardsController is RewardsDistributor, VersionedInitializable, IRewar
     uint256 totalSupply,
     uint256 userBalance
   ) external override {
-    _updateData(msg.sender, user, userBalance, totalSupply);
+    _updateData(user, msg.sender, userBalance, totalSupply);
   }
 
   /// @inheritdoc IRewardsController
