diff --git a/contracts/rewards/RewardsController.sol b/contracts/rewards/RewardsController.sol
index cfa4d82..ef3dedb 100644
--- a/contracts/rewards/RewardsController.sol
+++ b/contracts/rewards/RewardsController.sol
@@ -124,7 +124,7 @@ contract RewardsController is RewardsDistributor, VersionedInitializable, IRewar
     address reward
   ) external override returns (uint256) {
     require(to != address(0), 'INVALID_TO_ADDRESS');
-    return _claimRewards(assets, amount, msg.sender, msg.sender, to, reward);
+    return _claimRewards(assets, amount, msg.sender, msg.sender, msg.sender, reward);
   }
 
   /// @inheritdoc IRewardsController
