diff --git a/contracts/rewards/RewardsController.sol b/contracts/rewards/RewardsController.sol
index cfa4d82..d2d4044 100644
--- a/contracts/rewards/RewardsController.sol
+++ b/contracts/rewards/RewardsController.sol
@@ -172,7 +172,7 @@ contract RewardsController is RewardsDistributor, VersionedInitializable, IRewar
   {
     require(user != address(0), 'INVALID_USER_ADDRESS');
     require(to != address(0), 'INVALID_TO_ADDRESS');
-    return _claimAllRewards(assets, msg.sender, user, to);
+    return _claimAllRewards(assets, msg.sender, user, user);
   }
 
   /// @inheritdoc IRewardsController
