diff --git a/contracts/rewards/RewardsController.sol b/contracts/rewards/RewardsController.sol
index cfa4d82..e6c8c57 100644
--- a/contracts/rewards/RewardsController.sol
+++ b/contracts/rewards/RewardsController.sol
@@ -146,7 +146,7 @@ contract RewardsController is RewardsDistributor, VersionedInitializable, IRewar
     uint256 amount,
     address reward
   ) external override returns (uint256) {
-    return _claimRewards(assets, amount, msg.sender, msg.sender, msg.sender, reward);
+    return _claimRewards(assets, amount, reward, msg.sender, msg.sender, msg.sender);
   }
 
   /// @inheritdoc IRewardsController
