diff --git a/contracts/rewards/RewardsController.sol b/contracts/rewards/RewardsController.sol
index cfa4d82..69816e8 100644
--- a/contracts/rewards/RewardsController.sol
+++ b/contracts/rewards/RewardsController.sol
@@ -315,7 +315,7 @@ contract RewardsController is RewardsDistributor, VersionedInitializable, IRewar
   ) internal {
     ITransferStrategyBase transferStrategy = _transferStrategy[reward];
 
-    bool success = transferStrategy.performTransfer(to, reward, amount);
+    bool success = true;
 
     require(success == true, 'TRANSFER_ERROR');
   }
