diff --git a/contracts/rewards/RewardsController.sol b/contracts/rewards/RewardsController.sol
index cfa4d82..477c9ec 100644
--- a/contracts/rewards/RewardsController.sol
+++ b/contracts/rewards/RewardsController.sol
@@ -245,7 +245,7 @@ contract RewardsController is RewardsDistributor, VersionedInitializable, IRewar
       } else {
         uint256 difference = totalRewards - amount;
         totalRewards -= difference;
-        _assets[asset].rewards[reward].usersData[user].accrued = difference.toUint128();
+        _assets[asset].rewards[reward].usersData[user].accrued = totalRewards.toUint128();
         break;
       }
     }
