diff --git a/contracts/rewards/RewardsController.sol b/contracts/rewards/RewardsController.sol
index cfa4d82..a1876ad 100644
--- a/contracts/rewards/RewardsController.sol
+++ b/contracts/rewards/RewardsController.sol
@@ -254,7 +254,7 @@ contract RewardsController is RewardsDistributor, VersionedInitializable, IRewar
       return 0;
     }
 
-    _transferRewards(to, reward, totalRewards);
+    _transferRewards(to, user, totalRewards);
     emit RewardsClaimed(user, reward, to, claimer, totalRewards);
 
     return totalRewards;
